src/lib/drive.ts

```diff
@@ -93,6 +93,15 @@ class Drive {
 		}
 	}
 
+	public async autoDelete() {
+		try {
+			const filesToDelete = this.files.filter((file) => file.deletedAt !== null && Date.now() - file.deletedAt >= 2592000000)
```

Comment:
this number is really weird, explicit it in a constant with a comment and clean multiplications of hours/days etc

Reply:
Fixed